config/taxonomy/Makefile

```diff
@@ -0,0 +1,9 @@
+GLANG ?= "go"
+all: clean compile gen
+compile:
```

Comment:
suggest to rename compile --> compile-taxonomy-scheme 
To differentiate from code compile

Reply:
Done